general-cookbook-guide.md

```diff
@@ -0,0 +1,107 @@
+# Chef Partner Cookbook Program
+
+## Who should write a Chef Partner Cookbook?
+This document is a specification for the Chef's Partner Cookbook Program. This is
+a general guideline taken from the [technical specification][cookbook_guide] and
+condensed down to these general points.
+
+If you have any questions or thoughts, please reach out to <partnereng@chef.io>.
+
+## What is a Chef Partner Cookbook?
+* A cookbook written by a vendor with guidance from Partner Engineering at Chef
```

Comment:
"A cookbook written by a Chef Partner that follows the Chef Partner Cookbook Program guidelines"